### DISCUSSION TITLE
Theme override selection and cursor styling

### DISCUSSION THREAD

**Discussion by zambetti** [zed-industries/zed#8599] [Feature Requests]
To set my personal/single-player selection and cursor colors, I've read that the first item in `players` must be used. I've tried this in `experimental.theme_overrides`, but the selection and cursor colors do not change.

Is this feature supported as an override?

```
  "theme": "Ayu Dark",
  "experimental.theme_overrides":
  {
    "players":
    [ 
      {
        "cursor": "#F0F",
        "background": "#0F0",
        "selection": "#FF0"
      }
    ]
  },
  ...
```

**Comment by zambetti** [comment]
I've since made a theme in the `~/.config/zed/themes` folder and the `players` array to set the selection color works as expected. I suppose that I either made a mistake in my code in snippet above, or setting selection color does not currently work if in `theme_overrides`

**Reply by jasongitmail** [reply]
I'm unable to override the selection color too. I assume it doesn't work in `theme_overrides`.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
